camel-openmrs-fhir/src/test/java/org/openmrs/eip/fhir/routes/resources/ProcedureRouterTest.java

```diff
@@ -0,0 +1,163 @@
+package org.openmrs.eip.fhir.routes.resources;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.hamcrest.Matchers.notNullValue;
+import static org.openmrs.eip.fhir.Constants.HEADER_FHIR_EVENT_TYPE;
+import static org.openmrs.eip.fhir.Constants.PROCEDURE_ORDER_TYPE_UUID;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import org.apache.camel.Endpoint;
+import org.apache.camel.Message;
+import org.apache.camel.RoutesBuilder;
+import org.apache.camel.builder.AdviceWith;
+import org.apache.camel.builder.AdviceWithRouteBuilder;
+import org.apache.camel.builder.RouteBuilder;
+import org.apache.camel.component.mock.MockEndpoint;
+import org.apache.camel.test.spring.junit5.CamelSpringTestSupport;
+import org.apache.camel.test.spring.junit5.UseAdviceWith;
+import org.hl7.fhir.r4.model.ServiceRequest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+import org.openmrs.eip.fhir.FhirResource;
+import org.openmrs.eip.mysql.watcher.Event;
+import org.springframework.context.support.AbstractApplicationContext;
+import org.springframework.context.support.StaticApplicationContext;
+
+@UseAdviceWith
+@Disabled //TODO: {{openmrs.baseUrl}} is not being picked from application-test.properties
+class ProcedureRouterTest extends CamelSpringTestSupport {
```

Comment:
These tests need to be fixed before merging.

Reply:
Done